libs/feature/search/src/lib/state/effects.ts

```diff
@@ -130,8 +131,17 @@ export class SearchEffects {
               return of([state, favorites, null])
             }
             return this.filterGeometry$.pipe(
+              tap((geom) => {
+                const isValid = validGeoJson(geom)
+                if (!isValid) {
+                  throw '\nFilter geometry is not a valid GeoJson'
```

Comment:
It's better practice to throw `Error`s, not strings.

Although in that case I would throw an empty error and output the warning in the console.

Also I think `geojson-validation` provides traces for the error it found in the object, maybe that could be interesting to print as well to help debug an invalid geom? See https://gitlab.com/mjbecze/GeoJSON-Validation/-/tree/master?ref_type=heads#usage

```js
//test to see if `invalidFeature` is valid and return a "trace" which contains the error
const trace = gjv.isFeature(invalidFeature, true)
console.log(trace)
```